File: 백준/Bronze/5622. 다이얼/다이얼.py
alphabet_list = ["ABC", "DEF", "GHI", "JKL", "MNO", "PQRS", "TUV", "WXYZ"]

word = input()

word_list = list(word)
transform_word = []

for i in range(len(word_list)):
    for j in range(8):
        if word_list[i] in alphabet_list[j]:
            num_word = j + 2
            transform_word.append(num_word)

total_time = 0

for k in transform_word:
    total_time += 2 + (k-1) * 1

print(total_time)

File: 백준/Bronze/5622. 다이얼/다이얼.java
import java.io.*;
import java.util.Arrays;
import java.util.StringTokenizer;

public class Main {

    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        String[] alphaet_list = {"ABC", "DEF", "GHI", "JKL", "MON", "PQRS", "TUV", "WXYZ"};
        String input = br.readLine();

        char[] input_char = input.toCharArray();
        int total_time = 0;

        for (char c : input_char) {
            for (String s : alphaet_list){
                if (s.contains(String.valueOf(c))){
                    total_time += 2 + Arrays.asList(alphaet_list).indexOf(s) + 1;
//                    Arrays.asList(arr).indexOf("b")
                }
            }
        }
        sb.append(total_time);
        System.out.println(sb);
    }
           }

